medusa/server/web/config/post_processing.py

```diff
@@ -213,19 +220,13 @@ def isNamingValid(pattern=None, multi=None, abd=False, sports=False, anime_type=
             return 'invalid'
 
     @staticmethod
-    def isRarSupported():
-        """
-        Test Packing Support:
-            - Simulating in memory rar extraction on test.rar file
-        """
-
+    def isRarSupported(unrar_tool):
```

Comment:
@p0psicles 
This is really no excuse to keep using bad practices. Especially a route should **never** be camel case or even use upper case characters. I thought we wanted to enforce best practices and I don't see how this attitude is coherent with our goals.